### PR TITLE
UG update for Find text method in PDFViewer

### PR DESCRIPTION
**Pull request by YathavakrishnanMohan**
Description :
UG update for Find text method in PDFViewer

### CODE REVIEW

File: wpf/Pdf-Viewer/Searching-Text.md [hunk 1]
```diff
@@ -79,6 +79,8 @@ private void SearchPrevious_Click(object sender, RoutedEventArgs e)
 
 ## Find text method
 
+N> From version 21.1.x,a reference to the Syncfusion.PdfToImageConverter.Base assembly is necessary for PdfViewer FindText methods.We have updated the PDFBase approach to PdfToImageConverter approach for better performance.
```
Comment (Vikassekar) [inline wpf/Pdf-Viewer/Searching-Text.md:82]: As we discuss change the content based on the search text

**Review by Vikassekar — CHANGES_REQUESTED** [review]

File: wpf/Pdf-Viewer/Searching-Text.md (hunk 1 above)
Comment (YathavakrishnanMohan) [reply wpf/Pdf-Viewer/Searching-Text.md]: Change the content based on the search text is completed

File: wpf/Pdf-Viewer/Searching-Text.md [hunk 2]
```diff
@@ -79,6 +79,8 @@ private void SearchPrevious_Click(object sender, RoutedEventArgs e)
 
 ## Find text method
 
+N> From version 27.1.x, we have updated our default text extraction engine to PDFium for extracting text information from PDF documents. Based on the text information, we perform FindText in PDF documents. Please refer to the [link](https://help.syncfusion.com/wpf/pdf-viewer/text-extraction-engines) for more details.
```
Comment (Vikassekar) [inline wpf/Pdf-Viewer/Searching-Text.md:82]: Change as below 
From version 27.1.x, we have used text extraction engine for find text from PDF documents. By default text extraction engine uses PDFium for extracting text information. Please refer to the [link](https://help.syncfusion.com/wpf/pdf-viewer/text-extraction-engines) for more details

**Review by Vikassekar — CHANGES_REQUESTED** [review]

File: wpf/Pdf-Viewer/Searching-Text.md (hunk 2 above)
Comment (YathavakrishnanMohan) [reply wpf/Pdf-Viewer/Searching-Text.md]: Notes changes  completed

**Review by Vikassekar — APPROVED** [review]
approving this

**Pull request merged**